packages/core/src/plugin/js/index.ts

```diff
@@ -165,22 +169,54 @@ export function convertPlugin(plugin: JsPlugin): void {
     }
   }
 
+  if (plugin.renderResourcePot) {
+    plugin.renderResourcePot.filters ??= {};
+
+    if (!plugin.renderResourcePot.filters?.resourcePotTypes) {
+      plugin.renderResourcePot.filters.resourcePotTypes = [];
+    } else if (!plugin.renderResourcePot.filters?.paths) {
+      plugin.renderResourcePot.filters.paths ??= DEFAULT_FILTERS;
```

Comment:
`convertPlugin` is shared for Farm plugins and Vite plugins, we only need to set filters to `DEFAULT_FILTERS` for Vite plugins